lib/ruby_smb/signing.rb

```diff
@@ -12,8 +12,8 @@ module Signing
     # @param [RubySMB::GenericPacket] packet The packet to sign.
     # @return [RubySMB::GenericPacket] the signed packet
     def smb1_sign(packet)
-      packet = Signing::smb1_sign(packet, @session_key, @sequence_counter)
```

Comment:
The tests were mocking the `session_key` accessor, but not the `@session_key` instance variable. As a result, OpenSSL 3 was throwing validation errors that `''` was an invalid key.